modules/EnsEMBL/Web/Configuration/Variation.pm

```diff
@@ -124,6 +124,13 @@ sub populate_tree {
     { 'availability' => 'variation has_ldpops has_samples not_somatic', 'no_menu_entry' => 1 }
   );
 
+  if ($self->hub->species eq 'Homo_sapiens') {
```

Comment:
i dont quite understand the if check for human, does the availability check (is_coding) not make sure of this. Does is_coding return something for other species?

Reply:
We only have this mapping for Human so we don't want to show this icon/entry for non human species.
The `is_coding` only checks if the variant overlaps a coding exon (this could be used for other views in the future).